sklearn/decomposition/_lda.py

```diff
@@ -515,10 +515,7 @@ def partial_fit(self, X, y=None):
         # string, which is not the same one raised by check_n_features. So we
         # don't check n_features_in_ here for now (it's done with adhoc code in
         # the estimator anyway).
-        # TODO: set reset=first_time when addressing reset in
-        # predict/transform/etc.
-        reset_n_features = True
-        X = self._check_non_neg_array(X, reset_n_features,
+        X = self._check_non_neg_array(X, first_time,
```

Comment:
```suggestion
        X = self._check_non_neg_array(X, reset=first_time,
```

(I'd also suggest to rename `reset_n_features` into `reset` in `_check_non_neg_array`)


Also, should the comment above be removed:

> In theory reset should be equal to `first_time`...

`reset` *is* equal to `first_time` now,  it seems.